docs/guides/documentation/releases/README.md

```diff
@@ -0,0 +1,94 @@
+# Releases
+
+<pre align="center">Streamline software releases through consistent templates, automated aggregation and notifications.</pre>
+
+## Introduction
+
+**Background**: Making releases can be a chore. From ensuring the release notes have helpful information to aggregating many updates and notifying your users - doing the process right can be time consuming and fraught with misunderstandings. In this guide we recommend a standardized, automated way to notify your users of new releases. For example, we recommend using a `.github/release.yml` template to allow for more readable release notes, showcasing the importance of each update. We also provide automation for aggregating releases from many repositories into a super-release that provides a holistic view of project progress. Finally, we recommend automated notifications through GitHub integrations to help keep everyone informed.
+
+**Use Cases**:
```

Comment:
This looks like inline markup acting as a block heading—maybe change it to `###`?

[See also my Markdown Guidelines](https://github.com/NASA-PDS/nasa-pds.github.io/wiki/Markdown-Guidelines).

Reply:
Hey @nutjob4life - thanks for linking to your Markdown guidelines and suggestions! This format is based on the SLIM Guide Template available [here](https://nasa-ammos.github.io/slim/docs/contribute/submit-best-practice#use-our-standard-guide-template). I'm curious if you'd be interested in proposing an upgrade of the template based on your Markdown guidelines? That way we can apply best practice recommendations to all the guides (current and future) and not just this one.

Reply:
To be resolved at the template level, not here. See: https://github.com/NASA-AMMOS/slim/issues/161